analyses/molecular-subtyping-LGAT/input/snvOI_list.json

```diff
@@ -0,0 +1,101 @@
+{
+    "BRAF_V600E": [
+        {
+            "hotspot": 599,
+            "gene": "BRAF"
+        },
+        {
+            "hotspot": 600,
+            "gene": "BRAF",
+            "canonical": "p.V600E"
+        }
+    ],
+    "FGFR": [
+        {
+            "hotspot": 577,
+            "gene": "FGFR1"
+        },
+        {
+            "hotspot": 687,
+            "gene": "FGFR1"
+        },
+        {
+            "gene": "FGFR1",
+            "canonical": "p.N546K"
+        },
+        {
+            "gene": "FGFR1",
+            "canonical": "p.K656E"
+        }
+    ],
+    "MAPK": [
+        {
+            "gene": "KRAS"
+        },
+        {
+            "gene": "ARAF"
+        },
+        {
+            "gene": "NRAS"
+        },
+        {
+            "gene": "HRAS"
+        },
+        {
+            "gene": "MAP2K1"
+        },
+        {
+            "gene": "MAP2K2"
+        }
+    ],
+    "RTK": [
+        {
+            "gene": "KIT"
+        },
+        {
+            "gene": "MET"
+        },
+        {
+            "gene": "PDGFRA"
+        }
+    ],
+    "NF1": [
+        {
+            "gene": "NF1",
+            "cancer_disposition": "NF-1"
+        }
+    ],
+    "H3F3A": [
+        {
+            "canonical": "p.K28M",
+            "gene": "H3F3A"
+        },
+        {
+            "canonical": "p.G35R",
+            "gene": "H3F3A"
+        },
+        {
+            "canonical": "p.G35V",
+            "gene": "H3F3A"
+        }
+    ],
+     "HIST1H3B": [
```

Comment:
I might have forgotten to write this, but will you also add H3F3B to this list for H3.3?